src/pricing/calculations/extPaint.js

```diff
@@ -0,0 +1,35 @@
+import { floorsDiscount } from '../content/extPaint/categories'
+import { overHangRatio } from '../content/extPaint/categories'
+import { overHang } from '../content/extPaint/categories'
+import { windowPrice } from '../content/extPaint/categories'
+
+const parseDimensions = ({floors, homeSquareFoot, garageSquareFoot}) => {
```

Comment:
Some function, of course, are going to repeat in other categories (such as `parseDimensions`), so maybe we should put it in a `calculationTools.js` file or something